### PR TITLE
Bulk Domain Flow: Intro step

### PR DESCRIPTION
**Pull request by escapemanuele**
This PR updates the intro step in the Bulk Domain transfer flow

image

## Testing
1. Live link and visit ` /setup/bulk-domain-transfer/intro`
2. Check the page

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/bulk-domain-transfer-intro/intro.tsx [hunk 1]
```diff
@@ -19,25 +19,41 @@ const Intro: React.FC< Props > = ( { onSubmit } ) => {
...
 						description: (
-							<p>
-								{ __(
-									'Add all domain names with authorization codes to start transfer. Learn more'
-								) }
-							</p>
+							<>
+								<p>{ __( 'Add all domain names with authorization codes to start transfer.' ) }</p>
+								<a
+									href="wordpress.com"
```
Comment (alshakero) [inline client/landing/stepper/declarative-flow/internals/steps-repository/bulk-domain-transfer-intro/intro.tsx]: The link is incorrect.

**Review by alshakero — CHANGES_REQUESTED** [review]
Beautiful! Two tiny issues.

The heading is too big on mobile, it's ruining the centering.

<kbd>

</kbd>

**Review by alshakero — APPROVED** [review]
Looks 10/10 ✨ 

Besides linting issues, this is perfect.

**Comment by escapemanuele** [comment]
Thank you @alshakero , I'm gonna merge this as the lint issues are in the files Renan is working on now in https://github.com/Automattic/wp-calypso/pull/78847 👍

**Pull request merged**